src/features/seller/components/SellerConcertList.jsx

```
@@ -442,57 +490,84 @@ const SellerConcertList = ({
+                                                    <Edit size={16} />
+                                                </button>
+                                                <button
+                                                    onClick={() =>
+                                                        deleteConcert(concert)
+                                                    }
+                                                    className="p-1 text-red-400 hover:bg-red-900 hover:bg-opacity-30 rounded transition-colors"
+                                                    title="삭제"
+                                                >
+                                                    <Trash2 size={16} />
+                                                </button>
+                                            </div>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)